migrator.py

```diff
@@ -156,7 +290,20 @@ def migrate_face(image_id: int, filename: str, face_data: str, db: DigikamDb, co
     picasa_rect = rect64.parse_rect64(face_data[0])
     digikam_rect = rect64.to_digikam_rect(image_size, picasa_rect)
 
-    logging.debug('Adding face %s (%s) at %s to %s (%s)' % (tag_id, contact_id, digikam_rect, image_id, filename))
+    # Not sure if skipping an existing rect is an improvement or not... probably not since DigiKam creates rects at scan time
+    # and if the same rect is found in Picasa, it won't migrate.
+    # Should NOT skip if it is the first attempt at migration. however if you don't skip and identify the rect as someone else
+    # and then run this script again, you end up with another rect mapped to Picasa's name -- it's a conundrum..
+    if db.image_has_property(image_id, _FACE_TAG_REGION_PROPERTY, digikam_rect):
+        if skip_same_rect == None:
+            logging.error(f"digiKam already has face rectangle {digikam_rect} defined.  Please specify what to do by running with argument --skip_same_rect or --no-skip_same_rect")
+            assert(skip_same_rect is not None)
```

Comment:
This assert always fails. Use `raise ...` instead of logging.error() then, and no assert().

Reply:
OK